WaterLog_Backend/Models/MonitorsEntry.cs

```diff
@@ -12,7 +12,6 @@ public class MonitorsEntry
         public double Max_flow { get; set; }
```

Comment:
Quick suggestion, we can rename Max_flow to MaxFlow. But many of you are naming your variables according to the database field name. If thats the case then its cool.

Reply:
Yes this is the database field name.

Reply:
Sounds good, so this issue is sorted